Gordon360/Controllers/ProfilesController.cs

```diff
@@ -438,6 +439,22 @@ public async Task<ActionResult<StudentProfileViewModel>> UpdateMobilePhoneNumber
             return Ok(result);
         }
 
+        /// <summary>
+        /// Update office location (building description and room number)
+        /// </summary>
+        /// <param name="value">office location</param>
+        /// <returns></returns>
+        [HttpPut]
+        [Route("office_location/{value}")]
+        [AllowAnonymous]
```

Comment:
```suggestion

```
This route should only work for authenticated users, so the `[AllowAnonymous]` decorator should not be present.

Reply:
Thank you! Just removed that